src/NovelRT.Interop/DotNet/NrtRuntimeService.cpp

```diff
@@ -36,7 +36,17 @@ NrtResult Nrt_RuntimeService_initialise(NrtRuntimeService service) {
     }
 
     NovelRT::DotNet::RuntimeService* cService = reinterpret_cast<NovelRT::DotNet::RuntimeService*>(service);
-    cService->initialise();
+
+    try {
+      cService->initialise();
+    } catch (const Exceptions::InitialisationFailureException) {
+      Nrt_setErrMsgIsInitialisationFailureInternal();
+      return NRT_FAILURE_INITIALISATION_FAILURE;
+    } catch (const Exceptions::FunctionNotFoundException) {
```

Comment:
Nit: @RubyNova do we want the styling to be maintained like this? (Where each catch is inlined with bracing?)

Reply:
We should move to allman when you feel is appropriate.